schema/project-level/project-schema.json

```diff
@@ -10,7 +10,7 @@
   "properties": {
     "id": {
       "title": "Identifier or Reference",
-      "description": "A unique identifier or reference number for this infrastructure project, composed of a project identifier prefix and a local project identifier. For more information, see the [project identifier guidance](https://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/guidance/identifiers/#project-identifier-prefixes).",
+      "description": "An identifier for this infrastructure project. The identifier must be globally unique and must begin with a registered project identifier prefix. For more information, see the [project identifier guidance](https://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/reference/prefixes).",
```

Comment:
Hmm, this link is not "project identifier guidance", it's a list of prefixes. The guidance is still in the guidance section.

Reply:
Ah, yes. Good spot. Fixed in https://github.com/open-contracting/infrastructure/pull/355/commits/f1686d2efb0c77f13af0060f308efebf540e7e66